app/src/ui/history/expandable-commit-summary.tsx

```diff
@@ -221,22 +222,25 @@ export class ExpandableCommitSummary extends React.Component<
   }
 
   private renderExpander() {
-    if (
-      !this.state.body.length ||
-      (!this.props.isExpanded && !this.state.isOverflowed)
-    ) {
+    const { selectedCommits, isExpanded } = this.props
+    if (selectedCommits.length > 1) {
       return null
     }
 
-    const expanded = this.props.isExpanded
-    const onClick = expanded ? this.onCollapse : this.onExpand
-    const icon = expanded ? OcticonSymbol.fold : OcticonSymbol.unfold
-
     return (
-      <button onClick={onClick} className="expander">
-        <Octicon symbol={icon} />
-        {expanded ? 'Collapse' : 'Expand'}
-      </button>
+      <Button
+        onClick={isExpanded ? this.onCollapse : this.onExpand}
+        className="expander"
+        tooltip={isExpanded ? 'Collapse' : 'Expand'}
+        ariaExpanded={isExpanded}
```

Comment:
According to [the documentation](https://www.w3.org/TR/wai-aria/#aria-expanded) (and assuming I understand it correctly 😅 ), since the button doesn't _own_ the container that is being expanded but the other way around, I think this needs an `aria-controls="id-of-the-container"`. So you probably need to set an `id` again to this control after removing it in #17537

Reply:
Thanks for this! I did read this documentation and considered doing it, but I wanted to follow up with accessibility team because I know we have been given guidance on other components (mostly menu based so maybe that is the difference) and not asked to add the `aria-controls` attribute. I have it on my list to follow up on along with the zoom handling needed. Do you mind merging this for now knowing I do plan to follow up on this?

Reply:
Sounds good!!